geoh5py/shared/concatenation/object.py

```diff
@@ -162,7 +162,9 @@ def property_groups(self) -> list | None:
 
         return self._property_groups
 
-    def remove_children(self, children: list | Concatenated):
+    def remove_children(
```

Comment:
ok, we need the object, not the uids.

Still, why using "copy" above?

Reply:
see comment